public/app/features/dashboard/state/initDashboard.ts

```diff
@@ -226,7 +227,7 @@ export function getNewDashboardModelData(urlFolderId?: string | null): any {
       title: 'New dashboard',
       panels: [
         {
-          type: 'add-panel',
+          type: panelType ?? 'add-panel',
```

Comment:
This is all we need to support starting a new dashboard with an explicit panel type...  it may be a bit much, but kind nice to have:
* http://localhost:3000/dashboard/new?panelType=histogram&editPanel=1
* http://localhost:3000/dashboard/new?panelType=heatmap&editPanel=1
* http://localhost:3000/dashboard/new?panelType=graph&editPanel=1

as reliable links!